code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -22,6 +22,7 @@
 	head_icon = 'icons/mob/pai_item_head.dmi'
 	radio = /obj/item/radio/headset/silicon/pai
 	can_buckle_to = FALSE
+	mobility_flags = MOBILITY_FLAGS_DEFAULT | MOBILITY_REST
```

Comment:
Make a new define.

Reply:
I wanted to but I need a name for it that neither missleading nor too specific.

Reply:
Done.